src/Renderer.h

```diff
@@ -30,6 +36,14 @@ class Renderer
 		const unsigned int height = 800;
 		const unsigned int width = 800;
 
+		enum Scene
+		{
+			Test = 0,
+			PhysX = 1,
+		};
+
+		int sceneSelect;
```

Comment:
Just wondering what are these used for? I saw that you toggle `sceneSelect` when space is pressed but I don't see it being used for anything.
Maybe add a short comment if it is needed or remove it if it was just for some testing.

Reply:
I had a way to start the program in the original scene labelled "Test" to see everything working as it did before but wanted a quick way to switch to the physX scene.  Might be useful when we have Menu, inGame, Pause windows to run through

Reply:
Ok we can just leave it for now then.